testing/python/collect.py

```diff
@@ -841,7 +845,7 @@ def test_passed(x):
                 pass
         """
         )
-        result = testdir.runpytest()
+        result = testdir.runpytest(*ignore_parametrized_marks_args)
```

Comment:
those seem wrong, lets do a followup to make it nicer

Reply:
Why do you think they are wrong? They are using deprecated features, so I think it makes sense to explicitly mark those tests as such.

Reply:
i meant the way we pass those warning args, not the fact that we pass them

Reply:
Ahh OK. Do you have a suggestion on how to make them nicer? I don't mind doing that in this PR.

Reply:
i have no idea yet, there is a few options that may play differently into pytester and i'd like to have a while to let it fester at the back of my head as well

Reply:
OK, let's postpone this then.